scripts/test_docker.sh

```diff
@@ -61,6 +63,13 @@ fi
 declare server="https://${serverIP}:${port}"
 
 function finish {
+    if [ $interactive -eq 1 ]; then
+        echo "🤔 Do you want to stop the container? (Y/n)"
+        read -r proceed
+        if [ "$proceed" == "n" ]; then
+            exit 0
+        fi
+    fi
```

Comment:
If one does not stop the container to keep interacting with the network, how do they stop it later on?
I suppose we'll lose track of it's id and will end up with a dangling network.